[2025-09-19T15:00:03.220+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl_main.check_new_files scheduled__2025-09-19T14:45:00+00:00 [queued]>
[2025-09-19T15:00:03.234+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl_main.check_new_files scheduled__2025-09-19T14:45:00+00:00 [queued]>
[2025-09-19T15:00:03.235+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-09-19T15:00:03.271+0000] {taskinstance.py:2191} INFO - Executing <Task(BranchPythonOperator): check_new_files> on 2025-09-19 14:45:00+00:00
[2025-09-19T15:00:03.289+0000] {standard_task_runner.py:60} INFO - Started process 1529 to run task
[2025-09-19T15:00:03.298+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl_main', 'check_new_files', 'scheduled__2025-09-19T14:45:00+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmpgd88686u']
[2025-09-19T15:00:03.304+0000] {standard_task_runner.py:88} INFO - Job 102: Subtask check_new_files
[2025-09-19T15:00:03.408+0000] {task_command.py:423} INFO - Running <TaskInstance: crypto_etl_main.check_new_files scheduled__2025-09-19T14:45:00+00:00 [running]> on host ed735ff6f76b
[2025-09-19T15:00:03.574+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='crypto_etl_main' AIRFLOW_CTX_TASK_ID='check_new_files' AIRFLOW_CTX_EXECUTION_DATE='2025-09-19T14:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-19T14:45:00+00:00'
[2025-09-19T15:00:03.716+0000] {log.py:232} WARNING - [2025-09-19 15:00:03,715] INFO - crypto_etl - Listing files from bucket crypto-data with prefix 'raw-data/'
[2025-09-19T15:00:03.715+0000] {extract.py:26} INFO - Listing files from bucket crypto-data with prefix 'raw-data/'
[2025-09-19T15:00:03.730+0000] {log.py:232} WARNING - [2025-09-19 15:00:03,729] INFO - crypto_etl - Found 3 CSV files
[2025-09-19T15:00:03.729+0000] {extract.py:30} INFO - Found 3 CSV files
[2025-09-19T15:00:03.738+0000] {log.py:232} WARNING - [2025-09-19 15:00:03,737] INFO - crypto_etl - Found 0 unprocessed files out of 3 total
[2025-09-19T15:00:03.737+0000] {file_processor.py:25} INFO - Found 0 unprocessed files out of 3 total
[2025-09-19T15:00:03.739+0000] {logging_mixin.py:188} INFO - No new files to process
[2025-09-19T15:00:03.740+0000] {python.py:201} INFO - Done. Returned value was: no_new_files
[2025-09-19T15:00:03.741+0000] {branch.py:35} INFO - Branch into no_new_files
[2025-09-19T15:00:03.742+0000] {skipmixin.py:178} INFO - Following branch no_new_files
[2025-09-19T15:00:03.765+0000] {skipmixin.py:244} INFO - Skipping tasks [('validate_files', -1)]
[2025-09-19T15:00:03.836+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=crypto_etl_main, task_id=check_new_files, execution_date=20250919T144500, start_date=20250919T150003, end_date=20250919T150003
[2025-09-19T15:00:03.872+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-19T15:00:03.922+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
